doc/API.md

```diff
@@ -340,6 +340,15 @@ disregarded for forward compatibility.
 
 | Field          | Type   | Description                                         |
 | -------------- | ------ | --------------------------------------------------- |
+| `status`       | string | Filter the spend transactions by their [Spend status](#spend_status) |
+
+##### Spend status
+
+| Value           | Description                                                                                      |
+| --------------- | ------------------------------------------------------------------------------------------------ |
+| `non_final`     | The spend transaction is not final, we are awaiting signatures either from managers or cosigners |
+| `immature`      | We are awaiting for the `unvault_csv` to be able to broadcast the spend                          |
+| `broadcasted`   | The spend transaction has been broadcasted                                                       |
```

Comment:
Maybe also point here or in the introduction of the call that the status is only concerning the Spend wrt the signatures / Unvault timelock maturation and therefore they must refer to the status of each vault to know wether it was succesfully spent, canceled, or even emergencied